CounterStrike/src/dev/danablend/counterstrike/shop/Shop.java

```diff
@@ -150,6 +160,9 @@ public void purchaseShopItem(Player player, Weapon gun) {
         }
 
         if (CounterStrike.i.usingQualityArmory() && type != WeaponType.GRENADE && type != WeaponType.HELMET && type != WeaponType.ARMOUR) {
+            me.zombie_striker.qg.guns.Gun gun1 = me.zombie_striker.qg.api.QualityArmory.getGunByName(gun.getName());

```

Comment:
We have now hard dependency on QualityArmory, so absolute reference is not needed anymore, I think
```suggestion
            Gun gun1 = QualityArmory.getGunByName(gun.getName());

```